### PR TITLE
zip: new package at 3.0

### PR DESCRIPTION
**Pull request by AndreiSva**
## Description of package
Since the zip package from the main repository was dropped and lots of software here still depends on it I think it would be a good idea to bring it into community.

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/zip/version
/var/db/kiss/installed/zip/sources
/var/db/kiss/installed/zip/manifest
/var/db/kiss/installed/zip/depends
/var/db/kiss/installed/zip/checksums
/var/db/kiss/installed/zip/build
/var/db/kiss/installed/zip/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/zipsplit.1
/usr/share/man/man1/zipnote.1
/usr/share/man/man1/zipcloak.1
/usr/share/man/man1/zip.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/zipsplit
/usr/bin/zipnote
/usr/bin/zipcloak
/usr/bin/zip
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

### CODE REVIEW

**Comment by kyx0r** [comment]
Thanks, you beat me to it, on that notion, would you also package unzip ?

**Comment by AndreiSva** [comment]
We have busybox's unzip utility that *should* be enough for most things. Although if you ever need any features from info-ZIP's unzip then I'd be glad to package it :)

**Comment by kyx0r** [comment]
Well few days ago I had to extract some 10GB zip and busybox unzip failed, because it is bugged. I asked testuser for alternatives and infozip seemed to be the best choice. If you package it, I'll be neat. I wanted to do it myself afterall.

**Comment by AndreiSva** [comment]
Alright! I'll package it and make a new pr for that.

**Comment by AndreiSva** [comment]
@kyx0r #634

File: community/zip/version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.0 2
```
Comment (dilyn-corner) [inline community/zip/version:1]: 2 -> 1

**Pull request merged**